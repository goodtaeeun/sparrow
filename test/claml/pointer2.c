void f(char *p) { p[1] = 10; }
