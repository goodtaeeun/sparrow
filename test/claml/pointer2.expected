/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "pointer2.c"
void f(char *p) {

  {
#line 1
    *(p + 1) = (char)10;
    return;
  }
}
